modules/podyachikh_knight_moves_lab2/test/test_Podyachikh_knight_dist.cpp

```diff
@@ -0,0 +1,76 @@
+// Copyright 2024 Podyachikh Mikhail
+
+#include <gtest/gtest.h>
+
+#include "include/chess_board.h"
+
+TEST(testMinKnightMoves, obstacleExceptions) {
+  ChessBoard board = ChessBoard(12);
+  EXPECT_ANY_THROW(board.addObstacle({-1, 2}));
+  EXPECT_ANY_THROW(board.addObstacle({5, 12}));
+  EXPECT_NO_THROW(board.addObstacle({0, 11}));
+}
+
+TEST(testMinKnightMoves, knightExceptions) {
+  ChessBoard board = ChessBoard(12);
+  std::pair<int, int> available = {2, 5};
```

Comment:
```suggestion
  std::pair<int, int> available{2, 5};
```
Поправьте пожалуйста в других местах